sdk/core/azure-core/src/main/java/com/azure/core/util/CoreUtils.java

```diff
@@ -183,4 +197,44 @@ public static boolean isNullOrEmpty(CharSequence charSequence) {
         }
         return Collections.emptyMap();
     }
+
+    /**
+     * Pretty print the json or xml string content. If the content is not in a format of json or xml,
+     * return original string.
+     *
+     * @param content The body content which need to parse.
+     * @param contentType The format of the contents.
+     * @return Pretty json or xml format of the content. If it is not in a format of json or xml, returns original one.
+     */
+    public static String printPrettyFormatJsonOrXml(String content, String contentType) {
```

Comment:
This shouldn't be a public API. It should go into an implementation package utility location.

Reply:
Will create an ImplUtil under implementation pkg and put the function there.